### PR TITLE
Updating sidebar for mobile

### LINKED ISSUE #1726: Sticky header for mobile in benefits directory
https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5c3f8e799f82f718aea01b56
screen shot 2019-01-21 at 3 12 43 pm

### PR DESCRIPTION
**Pull request by SupeDeDupe**
Closes #1726 and #1754

### CODE REVIEW

**Comment by SupeDeDupe** [comment]
Need to set the offset in the scroll to only come into effect on mobile, then I think that'll be the end of the PR.

File: components/BB.js [hunk 1]
```diff
@@ -77,10 +101,21 @@ export class BB extends Component {
   }
   scrollToNextSteps() {
     window.location = "#next-steps";
+    const maxMobile = parseFloat(globalTheme.max.xs);
+    window.screen.width < maxMobile ? window.scrollBy(0, -90) : null;
```
Comment (smcmurtry) [inline components/BB.js:105]: I tested with my phone and this offset is working great! Can you apply it to the favourites page too?

Comment (SupeDeDupe) [reply components/BB.js:105]: Oh yes, I meant to not forget about that...

**Review by smcmurtry — APPROVED** [review]
Nice work! 🦑

**Pull request merged**